modules/@apostrophecms/doc-type/index.js

```diff
@@ -783,8 +784,10 @@ module.exports = {
             ...input
           };
         }
-
-        await self.apos.schema.convert(req, schema, input, doc);
+        const convertOptions = {
+          fetchRelationships: options.fetchRelationships
```

Comment:
we should enforce it to always be a boolean. If you pass `options` without `fetchRelationships` the value will be undefined and code later in this PR will not except this

Reply:
I think the cleaner would have been to choose an falsy option like: `noRelationshipFetch`
Because in the current state, the option can be null, so we have to check explicitly if the option passed is `false`. This is what I do when using the option. That's why I did not typed this one, because it might generates bugs.

Reply:
Updated to:
```javasccript
          fetchRelationships: options.fetchRelationships !== false
```